cincinnati/src/plugins/internal/metadata_fetch_quay.rs

```diff
@@ -99,6 +102,10 @@ impl QuayMetadataFetchPlugin {
 #[async_trait]
 impl InternalPlugin for QuayMetadataFetchPlugin {
     async fn run_internal(self: &Self, io: InternalIO) -> Fallible<InternalIO> {
+        get_tracer()
+            .get_active_span()
+            .update_name("metadata-fetch-quay".to_string());
```

Comment:
Use the predefined constant instead:
```suggestion
            .update_name(Self::PLUGIN_NAME.to_string());
```